homework/burgers/app.js

```diff
@@ -29,10 +33,18 @@ app.use(express.static(path.join(__dirname, "public")));
 app.get("/", index.home);
 app.get("/ingredients", index.inventory);
 app.get("/order", index.placeorder);
-// app.get("/kitchen", index.vieworders);
+app.get("/kitchen", index.vieworders);
+
+app.post("/newItem", index.additemPOST);
+app.post("/updateItem", index.updateitemPOST);
+app.post("/placeOrder", index.placeorderPOST);   // updates inventory based on order
+app.post("/placeOrder", index.placeorderPOST2);  // places order in database
```

Comment:
This is a weird way to separate two functions. If you want chaining behavior, link the functions more explicitly than this.